src/openrct2/world/Map.h

```diff
@@ -33,7 +33,7 @@ constexpr const int32_t MAXIMUM_MAP_SIZE_BIG = COORDS_XY_STEP * MAXIMUM_MAP_SIZE
 constexpr int32_t MAXIMUM_TILE_START_XY = MAXIMUM_MAP_SIZE_BIG - COORDS_XY_STEP;
 constexpr const int32_t LAND_HEIGHT_STEP = 2 * COORDS_Z_STEP;
 constexpr const int32_t WATER_HEIGHT_STEP = 2 * COORDS_Z_STEP;
-constexpr const int32_t MINIMUM_LAND_HEIGHT_BIG = MINIMUM_LAND_HEIGHT * COORDS_Z_STEP;
+constexpr const int32_t kMinimumLandHeight_BIG = kMinimumLandHeight * COORDS_Z_STEP;
```

Comment:
rather than Height if we call this `kMinimumLandZ` it will match the functions it is used with SetBaseZ and SetClearanceZ

Reply:
Is that for the variable kMinimumLandHeight_BIG or kMinimumLandHeight?

Reply:
kMinimumLandHeight_BIG